### PR TITLE
[MINOR][TEST]The unit test cases of "UnifiedMemoryManagerSuite" run failed

### PR DESCRIPTION
**Pull request by 10110346**
## What changes were proposed in this pull request?
OS:  Windows 7
When running `UnifiedMemoryManagerSuite`,  the following exception occurred:
`System memory 1000 must be at least 471859200. Please increase heap size using the --driver-memory option or spark.driver.memory in Spark configuration.
java.lang.IllegalArgumentException: System memory 1000 must be at least 471859200. Please increase heap size using the --driver-memory option or spark.driver.memory in Spark configuration.`

## How was this patch tested?
unit test

### CODE REVIEW

File: core/src/test/scala/org/apache/spark/memory/UnifiedMemoryManagerSuite.scala [hunk 1]
```diff
@@ -42,6 +42,7 @@ class UnifiedMemoryManagerSuite extends MemoryManagerSuite with PrivateMethodTes
       maxOffHeapExecutionMemory: Long): UnifiedMemoryManager = {
     val conf = new SparkConf()
       .set("spark.memory.fraction", "1")
+      .set("spark.testing", "true")
```
Comment (srowen) [inline core/src/test/scala/org/apache/spark/memory/UnifiedMemoryManagerSuite.scala]: This is already set for all tests by the test harness. Why is this related?

**Comment by HyukjinKwon** [comment]
**Before:**
Build started: [CORE] `org.apache.spark.memory.UnifiedMemoryManagerSuite` [PR-18289](https://ci.appveyor.com/project/spark-test/spark/branch/PR-18289)
Diff: https://github.com/apache/spark/compare/master...spark-test:PR-18289

**After:**
Build started: [CORE] `org.apache.spark.memory.UnifiedMemoryManagerSuite` [PR-18289](https://ci.appveyor.com/project/spark-test/spark/branch/C76F63EB-238B-4331-ABE5-F9D4C81C3BC8)
Diff: https://github.com/apache/spark/compare/master...spark-test:C76F63EB-238B-4331-ABE5-F9D4C81C3BC8

**Comment by HyukjinKwon** [comment]
```
[00:19:01] [info] UnifiedMemoryManagerSuite:
[00:19:02] [info] - single task requesting on-heap execution memory (797 milliseconds)
[00:19:02] [info] - two tasks requesting full on-heap execution memory (63 milliseconds)
[00:19:03] [info] - two tasks cannot grow past 1 / N of on-heap execution memory (172 milliseconds)
[00:19:03] [info] - tasks can block to get at least 1 / 2N of on-heap execution memory (313 milliseconds)
[00:19:03] [info] - TaskMemoryManager.cleanUpAllAllocatedMemory (326 milliseconds)
[00:19:03] [info] - tasks should not be granted a negative amount of execution memory (50 milliseconds)
[00:19:03] [info] - off-heap execution allocations cannot exceed limit (13 milliseconds)
[00:19:04] [info] - basic execution memory (360 milliseconds)
[00:19:04] [info] - basic storage memory (31 milliseconds)
[00:19:04] [info] - execution evicts storage (0 milliseconds)
[00:19:04] [info] - execution memory requests smaller than free memory should evict storage (SPARK-12165) (0 milliseconds)
[00:19:04] [info] - storage does not evict execution (0 milliseconds)
[00:19:04] [info] - small heap (0 milliseconds)
[00:19:04] [info] - insufficient executor memory (16 milliseconds)
[00:19:04] [info] - execution can evict cached blocks when there are multiple active tasks (SPARK-12155) (0 milliseconds)
[00:19:04] [info] - SPARK-15260: atomically resize memory pools (16 milliseconds)
```

It looks passed on Windows Server before this PR. Is this Windows 7 specific issue?

**Comment by 10110346** [comment]
@srowen @HyukjinKwon  I ran the unit tests in` intellij IDEA`, maybe that's the reason of my failure

**Comment by HyukjinKwon** [comment]
@10110346, let's close this for now if the reason for https://github.com/apache/spark/pull/18289#discussion_r121658995 can't be explained. I don't think that configuration is the reason for the failure and this passes on Windows Server.

Otherwise, if you strongly feel this is a Windows 7 specific issue, would you mind if I ask to run the same commands I ran in AppVeyor? If you are able to reproduce this, I will try that on Windows 7 too.

```
sbt -Pyarn -Phive -Phive-thriftserver clean package
sbt -Pyarn -Phive -Phive-thriftserver "test-only org.apache.spark.memory.UnifiedMemoryManagerSuite"
```

BTW, up to my knowledge, build on Windows requires `bash` (e.g., by installing Cygwin) in the path  due to several bash required processes (e.g., generating build info file).

**Comment by 10110346** [comment]
Ok, i will close it ,thanks @HyukjinKwon
